app/CocoaPods/CPAppDelegate.m

```diff
@@ -46,6 +48,16 @@ - (IBAction)installBinstubIfNecessary:(id)sender;
   [[self CLIToolInstallationController] installBinstub];
 }
 
+- (IBAction)showHomeWindow:(id)sender;
+{
+  if (self.homeWindowController == nil)
+  {
```

Comment:
Small style-nitpick, can you move the open curly to the same line as the `if`?

Love that you followed the semi-colons after the signatures in the implementation file, though! :+1: :)